testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -485,4 +480,245 @@ private void testSpecialCharacterColumnNamesWithColumnMappingMode(String mode)
             onDelta().executeQuery("DROP TABLE default." + tableName);
         }
     }
+
+    private void testSupportedPartitionedColumnMappingWrites(String columnMappingMode)
```

Comment:
No need to extract a method. The reason for existing helper test methods were OSS Delta Lake didn't support id column mapping in the old version. 

Also, please note that helper methods should be after the caller side.

Reply:
refactored. replaced with data generator.